packages/yew-test-runner/src/layout_tests.rs

```diff
@@ -0,0 +1,101 @@
+//! Snapshot testing of Yew components
+
+use yew::virtual_dom::VNode;
+use yew::{Html, Renderer};
+
+use crate::scaffold::{TestScaffold, TestScaffoldProps};
+
+#[derive(Debug)]
+pub struct TestLayout<'a> {
+    pub name: &'a str,
+    pub node: VNode,
+    pub expected: &'a str,
+}
+
+pub fn diff_layouts(layouts: Vec<TestLayout<'_>>) {
```

Comment:
Can you add documentation here for what `diff_layouts` does, steps it goes through and an example on how to use it?

Module level documentation for those who are unfamiliar with layout/snapshot testing would be good to have as well. We don't need to thoroughly explain what it is, a link to somewhere that explains it well alongside a short summary would be enough

Also, changing the signature to accept `IntoIterator` would be better.

```suggestion
pub fn diff_layouts<'a>(layouts: impl IntoIterator<Item = TestLayout<'a>>) {
```

This will allow the following to compile:
```rust
diff_layouts([layout]);
diff_layouts(vec![layout]);
```